config/locales/en.yml

```diff
@@ -127,6 +130,12 @@ en:
               invalid: Choose a CSV file in the correct format
         cohort_list_row:
           attributes:
+            submitted_at:
+              blank: "is required but missing"
+              invalid: "‘%{value}’ should be formatted as YYYY-MM-DDTHH:MM:SSZ"
```

Comment:
Does Excel have a name for this format? Or does it call it `YYYY-MM-DDTHH:MM:SSZ`? Just thinking that this may not mean anything to nurses in this format.

Reply:
Hopefully they don't edit this field, it's one of the ones that should be pre-filled by the download... The validation is more of a precaution.

Also, this field uses `Date.parse` which is quite lax on what it accepts, other formats should work fine.

That being said, we do ask them to input `YYYY-MM-DD` (ISO8601) for the date of birth, and this is the same format... but expanded to include exact time, as this will be used to disambiguate which are the first 100 parent records.